test/auth.e2e-spec.ts

```diff
@@ -108,10 +124,33 @@ describe('AuthController (e2e)', () => {
     });
   });
 
+  describe('GET /auth/activate-account', () => {
+    let token: string;
+    beforeEach(async () => {
+      const user = createUser();
+      const createdUser = await authService.signUp({
+        email: user.email,
+        password: user.password,
+      });
+      token = await authService.generateAccountActivationToken(createdUser.id);
+    });
+
+    it('should activate an account', async () => {
+      return request(app.getHttpServer())
+        .get(`/ auth / activate - account /? token = ${token}`)
```

Comment:
hmm, that's the first time I'm seeing such a thing 🤔 Is it the same as:

```
.get(`/auth/activate-account/?token=${token}`)
```
? Does it work?

Still, I think i would be better to omit the spaces so that it's not mistaken as some kind of comment 😛